frontends/web/src/routes/new-settings/components/device-settings/firmware-setting.tsx

```diff
@@ -42,7 +42,7 @@ const FirmwareSetting = ({ deviceID, versionInfo, asButton = false }: TProps) =>
   const [dialogOpen, setDialogOpen] = useState(false);
   const [confirming, setConfirming] = useState(false);
   const canUpgrade = versionInfo.canUpgrade;
-  const secondaryText = canUpgrade ? t('deviceSettings.firmware.upgradeAvailable') : t('deviceSettings.firmware.upToDate');
+  const secondaryText = canUpgrade ? t('deviceSettings.firmware.upgradeAvailable') : t('deviceSettings.deviceInformation.firmware.upToDate');
```

Comment:
not sure, but would it makes sense to make an exception for this one (as it is not a new key), so is stays  together next to `deviceSettings.firmware.upgradeAvailable` so that the translation is together with the other stuff in `deviceSettings.firmware`.

Reply:
Ahh sorry... we don't need to have this change here. Just edit on locize from "... Your device ..." to "... Your firmware ...". I'll revert this. 🙏